h2olog

```diff
@@ -627,6 +636,31 @@ int trace_quictrace_lost(struct pt_regs *ctx) {
 
     return 0;
 }
+
+// send_response_header(uint64_t conn_id, uint64_t req_id, const char *name, size_t name_len, const char *value, size_t value_len)
```

Comment:
Is this a memo that you were using, that you forgot to delete? (I do the same thing)

Reply:
Yes, it was just a memo. Removed 58a672b2714c9fe7a32a60126f916ad7cfed3db4